.github/workflows/CI.yaml

```diff
@@ -209,10 +209,12 @@ jobs:
       matrix:
         scheduling-gates: [gates_on, gates_off]
         allowed-namespaces: [allowed_ns_on, allowed_ns_off]
+        cert-manager-io-enabled: [cert-manager-io-on, cert-manager-io-off]
```

Comment:
shall we keep casing consistent and use `_` instead of `-` here?

Reply:
good catch, will adapt

Reply:
adapted